_includes/reactions.njk

```diff
@@ -0,0 +1,32 @@
+<div id="reactions-list">
+  <span id="big-smile" class="reaction"><a href="/reactions/big-smile/">😀</a></span>
+  <span id="smile" class="reaction"><a href="/reactions/smile/">🙂</a></span>
+  <span id="neutral" class="reaction"><a href="reactions/neutral/">😑</a></span>
+  <span id="frown" class="reaction"><a href="reactions/frown/">🙁</a></span>
+  <span id="angry" class="reaction"><a href="reactions/angry/">😡</a></span>
+</div>
+
+<script type="application/javascript">
+  const reactions = document.querySelectorAll(".reaction a")
+
+  for (let i = 0; i < reactions.length; i++) {
```

Comment:
You can use [for…of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#iterating_over_a_dom_collection) here.

Reply:
changed.. is there any benefit other than just looking better?